taxi/urls.py

```diff
@@ -55,28 +54,23 @@
         name="driver-detail"
     ),
     path(
-        "drivers/create",
+        "drivers/create/",
         DriverCreateView.as_view(),
         name="driver-create"
     ),
     path(
-        "drivers/<int:pk>/license_update",
+        "drivers/<int:pk>/license_update/",
         DriverLicenseUpdateView.as_view(),
         name="driver-update"
     ),
     path(
-        "drivers/<int:pk>/license_delete",
-        driver_license_delete,
-        name="driver-license-delete"
-    ),
-    path(
-        "drivers/<int:pk>/confirm_license_delete",
-        driver_confirm_delete,
+        "drivers/<int:pk>/confirm_license_delete/",
```

Comment:
you should add a driver deletion page not a license deletion page.

Reply:
its additional option. I immediately misunderstood condition)) then decided to keep this option in project. driver_deleter  also in the project

Reply:
If you go to your profile there will be a button "delete driver" , if you go to someone else’s profile there will be no button